README.md

```diff
@@ -3,6 +3,8 @@ About
 
 This site is the project website of the notification-app by the Dutch Ministry of Health.
 
+[![Build Status](https://travis-ci.org/jsoeterbroek/nl-covid19-notification-app-dashboard.svg?branch=patch-testing)](https://travis-ci.org/jsoeterbroek/nl-covid19-notification-app-dashboard)
```

Comment:
This should reference `minvws/nl-covid19-notification-app-website`

Reply:
I have taken the liberty to update this. Looking good! Merged 👍